packages/handbook-v1/en/Basic Types.md

```diff
@@ -172,45 +174,87 @@ enum Color {
 }
 let colorName: string = Color[2];
 
-console.log(colorName); // Displays 'Green' as its value is 2 above
+// Displays 'Green'
+console.log(colorName);
 ```
 
-# Any
+# Unknown
 
 We may need to describe the type of variables that we do not know when we are writing an application.
-These values may come from dynamic content, e.g. from the user or a 3rd party library.
-In these cases, we want to opt-out of type checking and let the values pass through compile-time checks.
-To do so, we label these with the `any` type:
+These values may come from dynamic content &ndash; e.g. from the user &ndash; or we may want to intentionally accept all values in our API.
+In these cases, we want to provide a type that tells the compiler and future readers that this variable could be anything, so we give it the `unknown` type.
 
 ```ts twoslash
-let notSure: any = 4;
+let notSure: unknown = 4;
 notSure = "maybe a string instead";
-notSure = false; // okay, definitely a boolean
+
+// OK, definitely a boolean
+notSure = false;
 ```
 
-The `any` type is a powerful way to work with existing JavaScript, allowing you to gradually opt-in and opt-out of type checking during compilation.
-You might expect `Object` to play a similar role, as it does in other languages.
-However, variables of type `Object` only allow you to assign any value to them. You can't call arbitrary methods on them, even ones that actually exist:
+If you have a variable with an unknown type, you can narrow it to something more specific by doing `typeof` checks, comparison checks, or more advanced type guards that will be discussed in a later chapter:
 
 ```ts twoslash
-// @errors: 2339
-let notSure: any = 4;
-notSure.ifItExists(); // okay, ifItExists might exist at runtime
-notSure.toFixed(); // okay, toFixed exists (but the compiler doesn't check)
+// @errors: 2322 2322 2322
+declare const maybe: unknown;
+// 'maybe' could be a string, object, boolean, undefined, or an array
+const aNumber: number = maybe;
+
+if (maybe === true) {
+  // TypeScript knows that maybe is a boolean now
+  const aBoolean: boolean = maybe;
+  // So, it cannot be a string
+  const aString: string = maybe;
```

Comment:
It is not immediately clear to me as a reader that one of these lines is an error, while the other is not an error.  Consider adding something to the comment to indicate that one of the lines will result in a compiler error while the other will not.

Side note: Would be totally cool if the TS docs page actually put the intellisense squiggles on these inline code samples so people could see the actual TSC errors right in the docs!

Reply:
It does, that's why I removed them from the comments: https://www.staging-typescript.org/docs/handbook/basic-types.html#tuple

Reply:
Oh man, so cool!